graylog2-web-interface/src/views/actions/SearchActions.ts

```diff
@@ -51,6 +53,7 @@ type SearchActionsType = RefluxActions<{
   refresh: () => Promise<void>,
   get: (searchId: SearchId) => Promise<SearchJson>,
   parameters: (parameters: (Array<Parameter> | Immutable.List<Parameter>)) => Promise<View>,
+  setFilterSearchTypes: (FilterSearchTypes) => Promise<void>,
```

Comment:
In my understanding the action name is misleading. While we are effectively filtering the executed search types at some point, the intention is to filter the search types of a _widget_ to be executed. The naming here implies that we need to pass _search type ids_, but judging from the code in `SearchStore`, we need to pass a list of _widget ids_ here.